lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -265,12 +271,14 @@ <h3 class="error-msgs" id="error-msg"></h3>
              url: "${section_data['sale_validation_url'] | n, js_escaped_string}",
              success: function (data) {
                  $('#error-msg').attr('class','success-msgs')
-                 $('#error-msg').html(data.message).show();
+                 edx.HtmlUtils.setHtml($('#error-msg'), data.message);
+                 $('#error-msg').show();
```

Comment:
@andy-armstrong: I think the following works:
```
edx.HtmlUtils.setHtml($('#error-msg'), data.message).show();
```
Would you prefer this over replicating the selector?  Or do you like this broken out?  If broken out, would you use a variable, or just leave it as is?  I'm just trying to learn JavaScript code style preferences.